Dockerfile

```
@@ -18,9 +18,7 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
-
 EXPOSE 8080
 
 # Run database migrations and start the Django application
-ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
+ENTRYPOINT sh -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"
```

```suggestion
ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"]
```
The Exec form is preferable because it works better with arguments that contain spaces or special characters.